.github/workflows/Push.yml

```diff
@@ -0,0 +1,31 @@
+name: Master Push
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT }}
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - uses: c-hive/gha-npm-cache@v1
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Format
```

Comment:
Does it mean no more commits directly to the master branch unless we run tests and linter?

Reply:
No, this file only runs when a PR is merged into to `master` so commits will still happen but formatting won't occur in the PR on push many times, only one time when the PR is merged!

Reply:
I mean, when you send a commit directly to the master without creating a PR, the CI will only do the formatting and skip the linting and testing steps because they are not included in the workflow. So my expectation is that we will only make changes through the PR based on these settings for CI.

Reply:
Exactly, remember the branch rules we have? Master can only be pushed to via PR now, even force pushes aren't allowed to master.

Reply:
That makes sense! Sorry, I didn't remember these settings, so I decided to clarify.